rdt-open-reader/src/main/java/io/ona/rdt/util/RDTSyncConfiguration.java

```diff
@@ -28,16 +36,41 @@ public int getSyncMaxRetries() {
 
     @Override
     public SyncFilter getSyncFilterParam() {
-        return BuildConfig.SYNC_FILTER_PARAM.equals(Constants.Config.TEAM)
-                ? SyncFilter.TEAM_ID : SyncFilter.PROVIDER;
+        return BuildConfig.SYNC_FILTER_PARAM.equals(Constants.Config.LOCATION)
+                ? SyncFilter.LOCATION_ID : SyncFilter.PROVIDER;
     }
 
     @Override
     public String getSyncFilterValue() {
         AllSharedPreferences sharedPreferences = RDTApplication.getInstance().getContext().userService().getAllSharedPreferences();
         String provider = sharedPreferences.fetchRegisteredANM();
-        return BuildConfig.SYNC_FILTER_PARAM.equals(Constants.Config.TEAM)
-                ? sharedPreferences.fetchDefaultTeamId(provider) : provider;
+        if (BuildConfig.SYNC_FILTER_PARAM.equals(Constants.Config.LOCATION)) {
```

Comment:
Same comments in this thread https://github.com/onaio/rdt-standard/pull/957/files#r613941069 apply here.